app/components/forms/login-form.js

```diff
@@ -109,6 +112,24 @@ export default Component.extend(FormMixin, {
       } catch (error) {
         this.get('notify').error(this.get('l10n').t(error.message));
       }
+    },
+    showPasswordLogin() {
+      if (pwShown === 0) {
+        pwShown = 1;
+        show();
+      } else {
+        pwShown = 0;
+        hide();
+      }
+      function show() {
+        $('#pwd').attr('type', 'text');
```

Comment:
Please do not use jquery for this. Just change attribute on the template based on property on the component. 

-- 

Eg. if a property `showPassword` is true, set attribute as text else password. And when they press on it use `this.toggleProperty` to toggle the value of it.